Week2/arrays.js

```diff
@@ -0,0 +1,8 @@
+var myArrys = [];
```

Comment:
It's better to use `let` or `const` instead of `var`, so can you change that:)?

Reply:
thank you, i changed var to let.